packages/core/src/browser/label-provider.ts

```diff
@@ -117,7 +142,7 @@ export class LabelProvider {
         if (!contrib) {
             return '';
         }
-        return contrib!.getLongName!(element);
+        return contrib.getLongName!(element);
     }
```

Comment:
This line change is unrelated to the PR.  Removing the unnecessary ! brings getLongName to match getIcon and getName.